### PR TITLE
feat: wmg query endpoint impl with filter dim terms

### PR DESCRIPTION
**Pull request by atolopko-czi**
Part 3 of 4 of [story](https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell-data-portal/1959).

Branched off of https://github.com/chanzuckerberg/single-cell-data-portal/pull/2008

### Reviewers
**Functional:** 
@MDunitz 

**Readability:** 
@tihuan 
### Reviewers
**Functional:** 

**Readability:** 

---

## Changes
* wmg query endpoint response now includes `filter_dimensions` section (except for `dataset` subsection, to be implemented next).

### CODE REVIEW

File: tests/unit/backend/wmg/api/test_v1.py [hunk 1]
```diff
@@ -168,5 +176,109 @@ def test__query__valid_request_returns_valid_response_body(
...
+                            {'ethnicity_ontology_term_id_2': 'ethnicity_ontology_term_id_2_label'}
+                        ],
+                        'sex_terms': [
+                            {'sex_ontology_term_id_0': 'sex_ontology_term_id_0_label'},
+                            {'sex_ontology_term_id_1': 'sex_ontology_term_id_1_label'},
+                            {'sex_ontology_term_id_2': 'sex_ontology_term_id_2_label'}
+                        ]
+                    }
             }
+            self.maxDiff = None
```
Comment (atolopko-czi) [inline tests/unit/backend/wmg/api/test_v1.py:223]: TODO: rm

Comment (atolopko-czi) [reply tests/unit/backend/wmg/api/test_v1.py:206]: did this on child branch

**Review by tihuan — APPROVED** [review]
API LGTM!

**Comment by atolopko-czi** [comment]
This PR subsumed by https://github.com/chanzuckerberg/single-cell-data-portal/pull/2068.

**Pull request closed without merging**